.gitattributes

```diff
@@ -0,0 +1 @@
+dist/* linguist-generated
```

Comment:
```suggestion
dist/** linguist-generated
```

Reply:
Bun lock file also

Reply:
You need to include all the generated files here.

Reply:
What generated files are there other than the lockfile and dist folder?

Reply:
Off hand I think thats it but the lockfile is not marked as generated.

Reply:
https://github.com/ubiquity-os-marketplace/daemon-disqualifier/pull/54/files#diff-618cd5b83d62060ba3d027e314a21ceaf75d36067ff820db126642944145393eR2

it is in the linguist file